### PR TITLE
Normalize serde_json imports

### LINKED ISSUE #138: Normalize serde_json imports

### PR DESCRIPTION
**Pull request by iinuwa**
Fixes #138.

### CODE REVIEW

File: src/r0/session/login.rs [hunk 1]
```diff
@@ -186,18 +180,16 @@ mod tests {
...
-                {
-                    "identifier": {
-                        "type": "m.id.user",
-                        "user": "cheeky_monkey"
-                    }
-                }
-                "#,
-            )
-            .unwrap(),
+            from_json_str::<UserInfo>(json).unwrap(),
```
Comment (iinuwa) [inline src/r0/session/login.rs:192]: I think this requires changes to the `Deserialize` impl in order for `from_json_value` to work.

Comment (jplatte) [reply src/r0/session/login.rs:192]: Why is that?

Comment (iinuwa) [reply src/r0/session/login.rs:192]: I was getting an error: expected borrowed string, or something. Didn't have time to investigate further.

Comment (jplatte) [reply src/r0/session/login.rs:192]: You just found a bug! :)

Try rebasing on top of master, that problem should be fixed.

Comment (iinuwa) [reply src/r0/session/login.rs]: Cool. That works now.

**Pull request merged**